### PR TITLE
state to install dev version

### PR DESCRIPTION
**Pull request by dlax**

### CODE REVIEW

**Comment by dlax** [comment]
@philpep can you have a look at this? (both httpredir and epel make travis fail :()

File: saemref/install.sls [hunk 1]
```diff
@@ -31,6 +31,29 @@ create-saemref-user:
...
+  pip.installed:
+    - name: setuptools
+    - ignore_installed: true
+
+install saem_ref from hg:
+  pip.installed:
+    - name: hg+http://hg.logilab.org/master/cubes/saem_ref#egg=cubicweb-saem_ref
+    - user: {{ saemref.instance.user }}
+    - install_options:
+      - --prefix=/home/{{ saemref.instance.user }}/local
```
Comment (philpep) [inline saemref/install.sls]: I'd use ``--user`` instead and let pip use the user virtualenv (kind of) in ``~/.local``. But I've no argument to choose between mine an your solution.

Comment (philpep) [inline saemref/install.sls]: Forget about this, I just see the code below and it better to have an explicit separated installation path here.

**Comment by philpep** [comment]
One thing that seems weird is that ``saemref:install:dev`` doesn't not use dev installation to create the instance (create, db-create, db-init) and does not run a dev instance.

But if it's only for testing/developping I'm ok with this.

File: saemref/install.sls (hunk 1 above)
Comment (dlax) [inline saemref/install.sls]: the thing is that I'd need to set `CW_CUBES_PATH` environment variable even when installed in user scheme, so there's no benefit.

**Comment by dlax** [comment]
Philippe Pepiot a écrit :
> One thing that seems weird is that |saemref:install:dev| doesn't not use
> dev installation to create the instance (create, db-create, db-init) and
> does not run a dev instance.
>

Done (but not tested) in 3bddc338113f095504dbc5698838c4f43e0601d8

**Comment by dlax** [comment]
Denis Laxalde a écrit :
> Philippe Pepiot a écrit :
>> One thing that seems weird is that |saemref:install:dev| doesn't not use
>> dev installation to create the instance (create, db-create, db-init) and
>> does not run a dev instance.
>>
>
> Done (but not tested) in 3bddc338113f095504dbc5698838c4f43e0601d8

Does not work...

**Comment by dlax** [comment]
Rebased on current master which includes #31.

**Pull request closed without merging**